opentelemetry-api/TrackDependency/src/main/java/com/example/TrackDependency.java

```diff
@@ -0,0 +1,55 @@
+package com.example;
+
+import com.azure.monitor.opentelemetry.exporter.AzureMonitorExporterBuilder;
+import io.opentelemetry.api.trace.Span;
+import io.opentelemetry.api.trace.SpanKind;
+import io.opentelemetry.api.trace.Tracer;
+import io.opentelemetry.context.Scope;
+import io.opentelemetry.sdk.OpenTelemetrySdk;
+import io.opentelemetry.sdk.trace.SdkTracerProvider;
+import io.opentelemetry.sdk.trace.export.BatchSpanProcessor;
+import io.opentelemetry.sdk.trace.export.SpanExporter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class TrackDependency {
+
+  private static final String CONNECTION_STRING = "<Your Connection String>";
+  private static final Logger logger = LoggerFactory.getLogger(TrackDependency.class);
+  private static final Tracer spanTracer = initTracer();
+
+  public static void main(String[] args) throws InterruptedException {
+    track();
+    Thread.sleep(6000); // wait at least 5 seconds
+  }
+
+  private static void track() {
+    Span span = spanTracer.spanBuilder("trackDependency").setSpanKind(SpanKind.CLIENT).startSpan();
```

Comment:
just so it's not confused that it needs to `trackDependency` is 

```suggestion
    Span span = spanTracer.spanBuilder("dependency name").setSpanKind(SpanKind.CLIENT).startSpan();
```